templates/zerver/unsubscribe_confirmation_prompt.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+<div class="modal hide" id="unsubscribe-confirmation" tabindex="-1" role="dialog"
+     aria-labelledby="unsubscribe-confirmation-label" aria-hidden="true">
+  <div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+  </div>
+  <form id="unsubscribe_confirmation_form" class="form-inline">
+      <div class="modal-body">
+        You are the only user in this stream. Are u sure u want to leave <span id="unsub_stream_name"></span>
```

Comment:
please use more formal english here, e.g. "are you sure you want to leave <name>?".